src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -81,7 +83,11 @@ const DomainList = (props) => {
       <TouchableNativeFeedback onPress={handleSetUnFollow}>
         <View style={styles.followContainer}>
           <View style={styles.buttonFollowing}>
-            <Text style={styles.textButtonFollowing}>{isCommunity ? 'Joined' : 'Following'}</Text>
+            {isFromUserFragment ? (
+              <MemoIc_senddm height={18} width={18} color={COLORS.gray410} />
+            ) : (
+              <Text style={styles.textButtonFollowing}>{isCommunity ? 'Joined' : 'Following'}</Text>
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering based on `isFromUserFragment` seems fine. However, it would be better to ensure that the `handleSetUnFollow` function behaves as expected when the `MemoIc_senddm` icon is displayed.

316:
Adding `isFromUserFragment` to PropTypes is good for type checking. However, as mentioned earlier, consider providing a default value for this prop in `DomainList.defaultProps`.
```diff
  DomainList.defaultProps = {
+   isFromUserFragment: false,
    item: {},
    isHashtag: false,
    isDomain: false,
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->